lib/pbench/test/unit/agent/task/test_make_result_tb.py

```diff
@@ -31,84 +31,54 @@ def config_and_logger(self):
     def test_bad_result_dir(self, result_dir, caplog):
         caplog.set_level(logging.ERROR, logger=self.logger.name)
         if result_dir:
-            expected_error_message = f"Invalid result directory provided: {result_dir}"
+            expected_error_message = "Invalid Result directory provided"
```

Comment:
You've got a lot of unit test failures; one is for this line, where the actual message has an argument value at the end, and since you're checking `.endswith` the fact that your expected message is embedded doesn't help.

Reply:
yeah right